packages/entity-browser/src/components/EntityBrowser/EntityBrowser.js

```diff
@@ -1,77 +1,21 @@
 import React from 'react'
-import * as ToccoUI from 'tocco-ui'
-import './styles.scss'
-
-export class EntityBrowser extends React.Component {
-  constructor(props) {
-    super()
-    props.initializeTable()
-  }
-
-  onOrderByChange = orderBy => {
-    this.props.setOrderBy(orderBy)
-  }
 
-  onPageChange = page => {
-    this.props.changePage(page)
-  }
+import SearchFormContainer from '../../containers/SearchFormContainer'
+import ListViewContainer from '../../containers/ListViewContainer'
 
-  cellRenderer = (values, record) => {
-    const formattedValues = values.map((v, idx) => (
-      <ToccoUI.FormattedValue key={idx} type={v.type} value={v.value}/>
-    ))
-
-    if (formattedValues.length > 0) {
-      return (
-        <span>
-          {formattedValues.reduce((prev, curr) => [prev, ', ', curr])}
-        </span>
-      )
-    }
-  }
+import './styles.scss'
 
-  render() {
-    const props = this.props
+export const EntityBrowser = props => {
+  props.initialize()
```

Comment:
Is it a good idea to call `initialize()` in the render function? Maybe componentWillMount/componentDidMount should be used?